serial_no: 31703351
cam_name: "cameraLeft"
yaml_url: "package://autorally_core/calibrations/cameraLeft.yaml"
auto_exposure: 1
exposure_time: 10000
rect_left: 0
rect_top: 0
rect_height: 1024
rect_width: 1280
image_data_format: "XI_RGB24"
trigger_source: 1
